src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -230,4 +230,64 @@
   <data name="VersionPropertyNotFound_DialogTitle" xml:space="preserve">
     <value>The SARIF specification recommends that the version property appear at the beginning of the log file. This log file does not conform to that recommendation.</value>
   </data>
+  <data name="ImportPREfastFilter" xml:space="preserve">
+    <value>PREfast log files (*.xml)|*.xml</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportStaticDriverVerifierFilter" xml:space="preserve">
+    <value>Static Driver Verifier log files (*.tt)|*.tt</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportFxCopFilter" xml:space="preserve">
+    <value>FxCop report and project files (*.xml)|*.xml</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportFortifyFilter" xml:space="preserve">
+    <value>Fortify log files (*.xml)|*.xml</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportFortifyFprFilter" xml:space="preserve">
+    <value>Fortify FPR log files (*.fpr)|*.fpr</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportCppCheckFilter" xml:space="preserve">
+    <value>CppCheck log files (*.xml)|*.xml</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportClangAnalyzerFilter" xml:space="preserve">
+    <value>Clang log files (*.xml)|*.xml</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportAndroidStudioFilter" xml:space="preserve">
+    <value>Android Studio log files (*.xml)|*.xml</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportPylintFilter" xml:space="preserve">
+    <value>Pylint log files (*.json)|*.json</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportTSLintFilter" xml:space="preserve">
+    <value>TSLint log files (*.json)|*.json</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportContrastSecurityFilter" xml:space="preserve">
+    <value>Contrast Security files (*.csf)|*.csf</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportLogOpenFileDialogTitle" xml:space="preserve">
+    <value>Import analysis log</value>
+    <comment>Used in the import log open dialog</comment>
+  </data>
+  <data name="ImportMSBuildFilter" xml:space="preserve">
+    <value>MSBuild files (*.xyz)|*.xyz</value>
```

Comment:
>xyz [](start = 35, length = 3)

@larrygolding@comcast.net  - what is the correct file extension for MSBuild?

Reply:
An MSBuild output file might be anything. I think it's `msbuild.log` by default, but you can specify any name on the command line. So the filter needs to be `*.*`.

---
In reply to: [506568586](https://github.com/microsoft/sarif-visualstudio-extension/pull/224#discussion_r506568586) [](ancestors = 506568586)